extensions/mssql/package.json

```diff
@@ -503,6 +503,11 @@
           "command": "mssql.designTable",
           "when": "connectionProvider == MSSQL && nodeType == Table && config.tableDesigner.enableFeature",
           "group": "0_query@3"
+        },
+        {
+          "command": "mssql.newTable",
+          "when": "connectionProvider == MSSQL && nodeType == Folder && nodeLabel == Tables && config.tableDesigner.enableFeature",
```

Comment:
what's the difference between the "Folder" and "Table" nodeType (5 lines above)?

Reply:
Oh, this is the "Tables" folder. Read too fast 😄